src/math/lgamma.rs

```diff
@@ -1,5 +1,6 @@
 use super::lgamma_r;
 
+/// The natural logarithm of the [Gamma function](https://en.wikipedia.org/wiki/Gamma_function) (f64).
```

Comment:
Minor nit since this line is a couple characters over 100 (same for f32)

```suggestion
/// The natural logarithm of the [Gamma function] (f64).
///
/// [Gamma function] https://en.wikipedia.org/wiki/Gamma_function
```